lua/ulx/modules/cl/xlib.lua

```diff
@@ -545,11 +547,12 @@ function xlib.makeslider( t )
 	pnl.TextArea.selectAll = t.selectall
 	pnl.Label:SizeToContents()
 	pnl:SetZPos( t.zpos or 0 )
+	if t.skin then pnl.Label:SetSkin( t.skin ) end
```

Comment:
```suggestion
	if t.skin then pnl:SetSkin( t.skin ) end
```

Applying a skin should take effect on the whole slider. This should also set the skin of the label, right? (it's been a while for me lol)